### PR TITLE
Fix- Changed the EcosystemTestTable toolTip content background-color to dark

### LINKED ISSUE #3208: [EcosystemTestTable]: Tooltip is using the wrong color
### Steps to reproduce the problem

1. Go to https://qiskit.org/ecosystem
2. Click on any card that has an `EcosystemTestTable` (dropdown accordion)
3. Hover over an of the `<Information>` icons to reveal the tooltip
4. See the tooltips using purple

### What is the current behavior?

The Tooltips have a purple background

### What is the expected behavior?

I believe they should have consistency with the other Tooltip in the card (using the default dark/black background)

### Screenshots

https://github.com/Qiskit/qiskit.org/assets/6276074/a9e70900-fc20-4456-b73e-f2aa137350ed

### Device

_No response_

### OS

_No response_

### Browser

_No response_

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by tarunsamanta2k20**
## Changes
[EcosystemTestTable]: Tooltip is using the wrong color

## Implementation details
Changed the EcosystemTestTable toolTip content background-color to dark
closes #3208 

## Screenshots

image

https://github.com/Qiskit/qiskit.org/assets/55488549/2cab9d30-0c17-4fc7-9b5c-3cf4251d065d

### CODE REVIEW

File: components/Ecosystem/EcosystemTestTable.vue [hunk 1]
```diff
@@ -129,4 +129,8 @@ bx-table-cell {
   background-color: carbon.$cool-gray-10;
   border-bottom: 1px solid carbon.$cool-gray-30;
 }
+bx-tooltip-icon {
+  --cds-inverse-02: #{carbon.$cool-gray-90};
+  line-height: 0;
```
Comment (techtolentino) [inline components/Ecosystem/EcosystemTestTable.vue:134]: @tarunsamanta2k20 - everything looks good, but just wondering if we need the `line-height` update?

**Review by techtolentino — APPROVED** [review]
Thank you @tarunsamanta2k20 - well done 👏🏼

**Review by techtolentino — APPROVED** [review]
Thank you @tarunsamanta2k20 - well done 👏🏼

**Comment by tarunsamanta2k20** [comment]
@techtolentino please add me as a member of your organization. I want to contribute more in the future.

**Comment by techtolentino** [comment]
hey @tarunsamanta2k20 

Thank you for wanting to continue contributing to the qiskit.org project — we appreciate your contributions so far, and it's nice to hear that you're interested in helping more.

Although we're unable to add you to our GitHub organization (it's an internally managed GitHub org), please know that we highly value your involvement as an external contributor.

We can always assign you tasks that align with your interests and skills, just tag us on any that you're interested in.

Again, thanks for your contributions!

**Pull request merged**